endpoints/multiple-versions/openapi-v2.yaml

```diff
@@ -0,0 +1,108 @@
+# [START swagger]
+swagger: "2.0"
+info:
+  description: "A simple Google Cloud Endpoints API example."
+  title: "Endpoints Example"
+  version: "2.0.0"
+host: "echo-api.endpoints.YOUR-PROJECT-ID.cloud.goog"
+# [END swagger]
+basePath: "/v2"
+consumes:
+- "application/json"
+produces:
+- "application/json"
+schemes:
+- "https"
+paths:
```

Comment:
I indented all the lists using https://github.com/OAI/OpenAPI-Specification/tree/master/examples/v2.0/yaml as examples. They seem to indent '-'. Also, if the item is more than one line, the '-' is on its own line.